pkgs/applications/science/math/mathics/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib, fetchFromGitHub, fetchPypi, buildPythonPackage, buildPythonApplication, isPy3k
+, glibcLocales, mpmath
+, numpy, pint, django_3, dateutil, llvmlite, requests, palettable, setuptools, wordcloud, scikitimage, python3
+, substituteAll
+}:
+
+let
+  sym = buildPythonPackage rec {
```

Comment:
It is necessary to have a separate version of sympy rather than using (or upgrading) the one in nixpkgs?

Reply:
Mathics requires sympy 1.7. But maybe I should indeed update sympy in nixpkgs.

Reply:
Done.